python/nav/models/logger.py

```diff
@@ -132,7 +132,7 @@ class MessageView(models.Model):
     Check: https://docs.djangoproject.com/en/dev/ref/models/options/
     """
 
-    origin = models.ForeignKey(
+    origin = models.OneToOneField(
```

Comment:
Here is an excellent reason for why the commit log needs to focus on **why**:

Why is this change being made? Why is this relevant for the JSONField PR?

Also, this doesn't sit with the data model as I know it...